src/applications/personalization/profile/tests/e2e/direct-deposit/update-flow.cypress.spec.js

```diff
@@ -36,8 +42,15 @@ function exitBankInfoForm() {
   cy.findByRole('button', { name: /cancel/i }).click();
 }
 
-function saveNewBankInfo() {
-  cy.findByRole('button', { name: /update/i }).click();
+function saveNewBankInfo(id) {
```

Comment:
Same deal here. We need a way to determine which form's UPDATE button we are clicking.